sync/04-docker-composer.sh

```diff
@@ -4,6 +4,25 @@ SCRIPT_COMPOSER_CACHE=${COMPOSER_CACHE_DIR:-$HOME/.cache/composer}
 
 mkdir -p ${COMPOSER_CACHE_DIR:-$HOME/.cache/composer}
 
+
+## T302558 Pre-installing composer-merge-plugin
+#
+# Because of https://github.com/wikimedia/composer-merge-plugin/issues/202
+# We are required to install the composer-merge plugin separately
+# This avoids the unwanted update that happens on a fresh install
+#
+# This means for now, the wikimedia/composer-merge-plugin version needs to be defined here
+COMPOSER_MERGE_PLUGIN_VERSION=v2.0.1
```

Comment:
This is slightly unfortunate but probably less work than trying to extract this from the lockfile